cypress/test_objects/buy-tshirt-objects.ts

```diff
@@ -0,0 +1,8 @@
+const userId = {
+  email: "aperdomobo@gmail.com",
+  password: "WorkshopProtractor",
+};
+
+const validationMessage = "Your order on My Store is complete.";
+
+export {validationMessage, userId};
```

Comment:
as this will only be used in one test, it does not makes sense to create a separated file with this configuration. Also, this makes hard to read what is the value we want to assert or verify in the tests.